src/damage.cpp

```diff
@@ -162,7 +162,7 @@ void ammo_effects( const tripoint &p, const std::set<std::string> &effects )
     if( effects.count( "NAPALM" ) > 0 ) {
         g->explosion( p, 4, 0.7, 0, true );
         // More intense fire near the center
-        for( auto &&pt : g->m.points_in_radius( p, 1, 0 ) ) {
+        for( auto && pt : g->m.points_in_radius( p, 1, 0 ) ) {
```

Comment:
Looks like we just need to use astyle version 2.05 or later, I used 2.04.something.
I'll re-run with a recenbt version of astyle and re-PR.